cogs/roles.py

```diff
@@ -141,6 +141,32 @@ async def inrole(self, ctx, *, queryRole):
                           editable_content=False)
             await pages.paginate()
 
+    @commands.command()
+    async def inchannel(self, ctx):
+        """Returns list of users in current channel"""
+        members = None
+        channel = ctx.message.channel
+        members = channel.members
+
+        if members is None:
+            return
```

Comment:
I don't think this could happen, you would get an empty list if there was no member, which anyway, isn't possible since presumably at least the bot and one user that calls the command are there

Reply:
Resolved in b20d0ac